src/main/java/nextstep/laddergame/domain/MemberLocation.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.laddergame.domain;
+
+public class MemberLocation {
+
+  private Member member;
+  private int location;
+
+  public MemberLocation(Member member, int location) {
+    if (member == null) {
+      throw new IllegalArgumentException();
+    }
+    if (location < 0) {
+      throw new IllegalArgumentException("위치는 0보다 작을 수 없습니다.");
+    }
+
+    this.member = member;
+    this.location = location;
+  }
+
+  public Point getPoint(Row row) {
+    return row.getPoint(location);
+  }
+
+  public MemberLocation goNextStep(Point point) {
+    if (point.checkLeftWay()) {
+      return new MemberLocation(member, location - 1);
+    } else if (point.checkRightWay()) {
+      return new MemberLocation(member, location + 1);
+    }
```

Comment:
```suggestion
    if (point.checkLeftWay()) {
      return left();
    } else if (point.checkRightWay()) {
      return right();
    }
```

극단적으로 메서드 분리한다면..ㅋㅋㅋ